theme6/source/stylesheets/categories.css

```diff
@@ -0,0 +1,350 @@
+.info-dialog {
+  display: flex;
+  align-items: center;
+  position: relative;
+  background-color: #abc828;
+  height: 35px;
+  justify-content: center;
+}
+
+.info-dialog p {
+  color: #ffffff;
+  font-family: 'Oswald', sans-serif;
+  font-size: 12px;
+  margin: 2px;
+}
+
+.dialog-button-container {
+  display: flex;
+  align-items: flex-start;
+  justify-content: flex-end;
+  position: absolute;
+  top: 7px;
+  right: 5px;
+  width: 100%;
+}
+
+.dialog-button-container button {
+  background-color: #ffffff;
```

Comment:
For hexadecimal codes containing the same letter (`f`) let's use only three of them instead of six